examples/spark/spark-worker-controller.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ReplicationController
+apiVersion: v1
+metadata:
+  name: spark-worker-controller
+  labels:
+    component: spark-worker
```

Comment:
Your other replication controllers have the `-controller` suffix on the component (e.g. `spark-master-controller` and `spark-driver-controller`). I don't have a preference for either, I'm just suggesting that they be consistent.